docs-next/components/Header.tsx

```diff
@@ -0,0 +1,168 @@
+/** @jsx jsx  */
+import { jsx } from '@keystone-ui/core';
+import Link from 'next/link';
+
+export function Header() {
+  return (
+    <header
+      css={{
+        borderBottom: '1px solid var(--gray-200)',
+      }}
+    >
+      <div
+        css={{
+          display: 'grid',
+          gridTemplateColumns: 'auto 1fr',
+          justifyItems: 'start',
+          padding: '1rem',
+          maxWidth: '66rem',
+          margin: '0 auto',
+        }}
+      >
+        <h2>
+          <Link href="/" passHref>
+            <a
+              css={{
+                backgroundImage: 'linear-gradient(to right, var(--blue-500), var(--indigo-600))',
+                backgroundClip: 'text',
+                fontWeight: 600,
+                fontSize: '1.25rem',
+                lineHeight: '1.75rem',
+                color: 'transparent',
+                verticalAlign: 'middle',
+                transition: 'color 0.3s ease',
+              }}
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 220 220"
+                css={{
+                  display: 'inline-block',
+                  width: '2rem',
+                  height: '2rem',
+                  margin: '0 0.5rem 0.25rem 0',
+                  verticalAlign: 'middle',
+                }}
+              >
+                <defs>
+                  <linearGradient id="logo-a" x1="0%" x2="50%" y1="0%" y2="71.9%">
+                    <stop offset="0%" stopColor="#5AE8FA" />
+                    <stop offset="100%" stopColor="#2684FF" />
+                  </linearGradient>
+                </defs>
+                <path
```

Comment:
Not a blocker, but I think this would be easier to read if the svgs were put in their own components (even if not in separate files, just naming them rather than inlining them here would make the component easier to scan)

Reply:
**edit** noticed later in the review that these were inlined previously as well, worth scouting anyway

Reply:
Yeah I agree. I didn't want to fix too much in this already huge PR. There are other thing here as well that I want to do including aligning the blue with our theme.